tensorflow/lite/micro/kernels/xcore/xcore_interpreter.cc

```diff
@@ -11,10 +12,12 @@ XCoreInterpreter::XCoreInterpreter(const tflite::Model* model,
                                    tflite::MicroAllocator* allocator,
                                    tflite::ErrorReporter* reporter,
                                    bool use_current_thread,
+                                   MemoryLoader& loader,
                                    tflite::MicroProfiler* profiler)
     : tflite::MicroInterpreter(model, resolver, allocator, reporter, profiler),
       dispatcher_(reporter, use_current_thread) {
   SetDispatcher(&dispatcher_);
+  SetMemoryLoader(std::addressof(loader));
```

Comment:
Unless `MemoryLoader` is expected to overload the `&` operator, we should stick to
```suggestion
  SetMemoryLoader(&loader);
```